include/revng/TupleTree/TupleTree.h

```diff
@@ -219,7 +219,7 @@ class TupleTree {
   void initializeUncachedReferences() {
     TrackGuard Guard(*Root);
     visitReferences([this](auto &Element) {
-      Element.Root = Root.get();
+      Element.setRoot(Root.get());
```

Comment:
There's a spurious `\` in the commit message

Reply:
Fixed.